package com.course.bloco_de_notas.services.exceptions;public class DataBaseException {
}
